src/main/java/use_cases/monthly_menu/MonthMenuOD.java

```diff
@@ -70,28 +70,4 @@ public String getWarning() {
     public boolean isSuccessful() {
         return successful;
     }
-
-    /**
-     * Sets specified ArrayList of Expense to expenseData variable.
-     * @param expenseData ArrayList of Expense stored in a MonthlyStorage
-     */
-    public void setExpenseData(ArrayList<Expense> expenseData) {
```

Comment:
I think I'll still need setters to update the list from adds and edits. Maybe I'm wrong?

Reply:
You shouldn't be needing these setters, because firstly, they are not original list of Expense/Category stored in MonthlyStorage; secondly, you are probably adding/editing the MonthlyStorage in your use case interactor.

Reply:
YES